src/components/Cart/Add-To-Cart.jsx

```diff
@@ -34,7 +34,7 @@ export const AddToCart = ({ product, btnNormalCss, btnSuccessCss }) => {
                 type="button"
                 disabled={!product.inStock}
                 className={isItemInCart ? btnNormalCss : btnSuccessCss}
-                onClick={() => AddToCartHandler}
+                onClick={() => AddToCartHandler()}
             >
```

Comment:
* You can just pass the function name here without the arrow function.
* Also you don't need PR for minor fix

Reply:
Ok Thnaks